correlation.py

```
@@ -512,7 +714,13 @@ def generate_report(
                 html += f"<p>{_html.escape(text_str)}</p>"
         
         elif element_type == 'interpretation':
-            html += f"<div class='interpretation'>{_html.escape(str(data))}</div>"
+            html += f"<div class='interpretation'>📊 {_html.escape(str(data))}</div>"
+        
+        elif element_type == 'summary':
+            html += f"<div class='summary-box'>{str(data)}</div>"
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)